tools/build_pak_files.py

```diff
@@ -0,0 +1,135 @@
+                "-stdout",
+                "-FullStdOutLogOutput",
+                "-CrashForUAT",
+                "-unattended",
+                "-NoLogTimes",
+                "-UTF8Output"
+            ]
+            print(f"[SPEAR | build_pak_files.py] Executing: {' '.join(cmd)}")
+            subprocess.run(cmd, check=True)
+
+            platform_dir = os.path.realpath(os.path.join(unreal_project_dir, "Saved", "Cooked", f"{platform}NoEditor"))
+            content_dirs_for_pak = [
+                os.path.join(platform_dir, "Engine", "Content", "Animation"),
+                os.path.join(platform_dir, "Engine", "Content", "BasicShapes"),
+                os.path.join(platform_dir, "Engine", "Content", "EngineResources"),
+                os.path.join(platform_dir, "Engine", "Content", "Functions", "Engine_MaterialFunctions02", "Math"),
+                os.path.join(platform_dir, "Engine", "Content", "Functions", "Engine_MaterialFunctions02", "Texturing"),
+                os.path.join(platform_dir, "Engine", "Content", "Functions", "Engine_MaterialFunctions02", "Utility"),
+                os.path.join(platform_dir, "Engine", "Content", "Functions", "MaterialLayerFunctions"),
+                os.path.join(platform_dir, "SpearSim", "Content")
+            ]
+
+            # create the output_dir
+            os.makedirs(args.output_dir, exist_ok=True)
+
+            pak_file_prefix = f"{scene_name}_{platform}"
+
+            # construct path to the output pak file
+            pak_file = os.path.join(args.output_dir, pak_file_prefix + ".pak")
+
+            # text file used to generate the final pak file
+            txt_file = os.path.join(args.output_dir, pak_file_prefix + ".txt")
+
+            for i, dir in enumerate(content_dirs_for_pak):
+                with open(txt_file, mode="w" if i==0 else "a") as f:
+                    for content_file in glob.glob(os.path.join(dir, "**", "*.*"), recursive=True):
+                        assert content_file.startswith(platform_dir)
+                        content_file = content_file.replace('\\', "/")
+                        mount_file = posixpath.join("..", "..", f"..{content_file.split(platform + 'NoEditor')[1]}")
+                        f.write(f'"{content_file}" "{mount_file}" "" \n')
+
+            # command to generate the final pak file
+            cmd = [
+                unreal_pak_bin,
+                pak_file,
+                "-create=" + txt_file,
+                "-platform=" + platform,
+                "-multiprocess",
+                "-compress"
+            ]
+            print(f"[SPEAR | build_pak_files.py] Executing: {' '.join(cmd)}")
+            subprocess.run(cmd, check=True)
+
+            assert os.path.exists(pak_file)
+            print(f"[SPEAR | build_pak_files.py] Successfully built {pak_file} for {platform} platform.")
+
+            print(f"[SPEAR | build_pak_files.py] Removing symlink {unreal_project_content_scene_dir}")
```

Comment:
`Removing symlink:` to match `Creating symlink:` above.

Reply:
Done